ultimarc/tools/mini_pac.py

```diff
@@ -16,7 +16,7 @@
 # Tool_cmd and tool_desc name are required.
 # Remember to add/update bash completion in 'tools.bash'
 tool_cmd = _('mini-pac')
-tool_desc = _('manage mini-pac devices')
+tool_desc = _('Manage Mini-pac devices')
 
```

Comment:
I usually leave all the tool descriptions lowercase, but if you want to change that I'm good with that.  Probably should make all the tool descriptions the same case style.